libqtile/core/manager.py

```diff
@@ -311,7 +311,7 @@ def finalize(self) -> None:
         self.core.finalize()
 
     def _process_screens(self, reloading=False) -> None:
-        current_groups = [s.group for s in self.screens if hasattr(s, "group")]
+        current_groups = [s.group for s in self.screens() if hasattr(s, "group")]
```

Comment:
This was the point you were referring to with `qtile.screens` and `qtile.groups`.

If you don't want to change the property name then we can change the exposed method name. I would assume this is only really relevant for tests.

Reply:
> I would assume this is only really relevant for tests.

I agree and hope so. I would guess that more people would be accessing `qtile.screens` in custom code than would be doing `qtile cmd-obj -o cmd -f screens` in some scripts.

What would you think about naming the commands `get_screens` etc?

Reply:
Yeah - this is probably a better approach. Can do the same for groups.

Reply:
Have pushed an update for that.